### ISSUE TITLE
Appendix Hyperrefs Point to Chapters

### ISSUE THREAD

**Issue by shanemcq18** [shanemcq18/utexas-thesis-template#3]
Clicking a link to an appendices sends the reader to a chapter in the body instead. Do we need a separate counter somewhere for appendices to fix this?

**Comment by shanemcq18** [comment]
Possible solution: https://tex.stackexchange.com/questions/433368/hyperlinks-to-toc-from-chapter-in-appendix

**Comment by shanemcq18** [comment]
The only outstanding problem is referring to sections of an appendix points to the body. Example:

```latex
\appendices
\chapter{My Appendix}

\section{My Section}
\label{sec:appendix:mysection}
```

Referring to `mysection` with `\cref{mysection}` will point you to section 1 of chapter 2 in the body.

**Comment by shanemcq18** [comment]
I think this has been fixed by d36be7d014dcbe68211596e1dd135b9c816f0a5f. Other issues may be fixed by adjusting the `hyperref` import:

```latex
\usepackage[hypertexnames=false]{hyperref}
```

https://tex.stackexchange.com/questions/424686/crefrange-links-to-the-right-line-in-the-wrong-algorithm
